pulpcore/cli/file/repository.py

```diff
@@ -82,6 +83,7 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
 repository.add_command(task_command(decorators=nested_lookup_options))
 repository.add_command(version_command(decorators=nested_lookup_options))
 repository.add_command(label_command(decorators=nested_lookup_options))
+repository.add_command(repository_content_command(contexts={"file": PulpFileContentContext}))
```

Comment:
Is there nothing in the code to prevent this? I know the plugin's repository models has a `CONTENT_TYPES` list field, but I'm not sure if that's check when adding content. Going with that naming scheme could be a lot of extra typing for plugin's with more than one content type and I'm not sure if there's benefit for allowing cross-plugin content sharing.

Reply:
We can probably add that later...if needed
Of course we would add the "file:" by default for the user.